### PR TITLE
Move some canteens to either leg

### PR DESCRIPTION
**Pull request by mugling**
Many of the canteens are unlikely to see significant usage (especially the hip flask) because they compete with the `WAIST` layer of the `TORSO`. These items would naturally be worn from the waist extending down the leg so this PR updates them to be cover `LEG_EITHER`. The volume of the canteen is reduced somewhat for balance.

The larger canteens remain unchanged as they clearly have their place (especially the jerrypack) for the bulk transport of liquids without the use of a vehicle.

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Don't change the volumes; they're supposed to be those sizes because those are standard sizes for canteens.

**Comment by mugling** [comment]
@Rivet-the-Zombie Noted, fixed in 5789f20c5f

File: data/json/items/containers.json [hunk 1]
```diff
@@ -181,7 +181,7 @@
         "color": "green",
         "name": "plastic canteen",
         "category": "other",
-        "description": "A large military-style water canteen, with a 1.5 liter capacity and carrying strap.",
+        "description": "A military-style water canteen with a 1.5 liter capacity. Commonly worn at the hip.",
```
Comment (Rivet-the-Zombie) [inline data/json/items/containers.json]: Double space after sentence breaks.

**Comment by Rivet-the-Zombie** [comment]
Please swap the encumbrance on the canteen and waterskin; the canteen was custom-engineered to be easy to wear, while the waterskin is held on by a strap.

**Comment by MehMuffin** [comment]
So does leg_either remove them from the torso waist area competition entirely, or just allow for two things in it at once? Could these be worn with a belt without penalties or not, that is?

**Pull request merged**